docs/src/content/docs/book/exit-codes.mdx

```diff
@@ -644,7 +648,9 @@ try {
 
 ### 137: Masterchain support is not enabled for this contract {#137}
 
-Any attempts to point to masterchain (ID $-1$) or otherwise interact with it without [enabling masterchain support](/book/masterchain#support) throw an exception with exit code $137$: `Masterchain support is not enabled for this contract`.
+<Badge text="Removed since Tact 1.6" variant="tip" size="medium"/><p/>
+
+Any attempts to point to masterchain (ID $-1$) or otherwise interact with it without enabling masterchain support throw an exception with exit code $137$: `Masterchain support is not enabled for this contract`.
```

Comment:
```suggestion
Prior to removel, nny attempts to point to masterchain (ID $-1$) or otherwise interact with it without enabling masterchain support were throwing an exception with exit code $137$: `Masterchain support is not enabled for this contract`.
```